### PR TITLE
Added disabling Bing & automatic File Explorer folder type discovery tips

### PR DESCRIPTION
**Pull request by itsWindows11**

### CODE REVIEW

**Comment by FloatingMilkshake** [comment]
Looks good to me, though I would maybe include a note about backing up the registry—messing with the registry can be dangerous, and it would be helpful to direct people to make a backup before they change anything. That way, if something goes wrong, they can easily roll back.

**Comment by itsWindows11** [comment]
Do I include this in each section (i.e. one for disabling Bing and another for FE) or just at the beginning of the article?

**Comment by sthivaios** [comment]
> Do I include this in each section (i.e. one for disabling Bing and another for FE) or just at the beginning of the article?

Imo it might be better to add a little section on how to do that at the bottom of the page and then add a link in each section that takes you to the registry backup section. You can of course do that with headers.

**Comment by FloatingMilkshake** [comment]
I don't feel that backing up the registry should be its own section, so I would say a quick mention in each section works. Maybe something along the lines of

> [vuepress note]
> 
> **Important!**
> Before making any changes to the registry, you should take a backup of its current state. You can back up the [folder name] folder by right-clicking it, selecting **Export**, and choosing a safe place like your Desktop to store the backup. If something goes wrong in this section, you can double-click the file on your desktop to undo your changes to that part of the registry.
> 
> [maybe include a screenshot]

**Comment by sthivaios** [comment]
> I don't feel that backing up the registry should be its own section, so I would say a quick mention in each section works. Maybe something along the lines of
> 
> > [vuepress note]
> > **Important!**
> > Before making any changes to the registry, you should take a backup of its current state. You can back up the [folder name] folder by right-clicking it, selecting **Export**, and choosing a safe place like your Desktop to store the backup. If something goes wrong in this section, you can double-click the file on your desktop to undo your changes to that part of the registry.
> > [maybe include a screenshot]

fair, but i would make it a caution or warning, not a note

**Comment by FloatingMilkshake** [comment]
> fair, but i would make it a caution or warning, not a note

I agree; by "note" I just meant it would be useful to use one of the fancy vuepress note things—I don't know the proper name for them. Apologies for the lack of clarity. Backups (especially of the registry!) are very important, and they should be stressed here because messing with the registry can be dangerous! I think the 'danger' type with an "IMPORTANT" title or something like that would work. 'Warning' and 'danger' are both probably fine though.

**Comment by itsWindows11** [comment]
> I just meant it would be useful to use one of the fancy vuepress note things

Custom containers: https://vitepress.dev/guide/markdown#custom-containers

It also supports expand/collapse details too. There are other markdown extensions listed in that page but no idea whether VitePress supports additional markdown extensions compared to VuePress 2 that we may want to use, anyway probably offtopic for this PR :p

**Review by Erisa — APPROVED** [review]

File: wiki/useful-windows-tips.md [hunk 1]
```diff
@@ -40,8 +40,45 @@ The text in front of the dot has no effect. You may change it to whatever you wa
...
+There are a few ways you can make Windows Search faster and make it search for your apps and files instantly.
+
+#### Disabling Bing Search on Windows 11/10
+
+By default, when you search for anything on your device, it gets searched on Bing, most of the time unintentionally, and you want an on-device result instead. This can slow down performance especially with excessive keystrokes. To disable this behavior you can follow the steps below:
+
+1. Search for "Registry Editor" in the Start menu.
+
+![Screenshot of the user searching for the Registry Editor in the Start menu search.](./img/useful-windows-tips/regedit-search-result.png)
+
```
Comment (sthivaios) [inline wiki/useful-windows-tips.md:55]:
```suggestion

::: danger Caution
Before making changes to the registry, you should always backup the current state of the folder you are modifying. Right click on the folder you are making the changes to, in the Registry Editor, then click **Export** and save the file somewhere safe. If something goes wrong, you can restore the backup by double-clicking on the file. You can also go to **File > Import**, in the Registry Editor.
:::

```

File: wiki/useful-windows-tips.md [hunk 2]
```diff
@@ -166,6 +203,38 @@ You will then find the keyboard icon on the tray area of the taskbar.
...
+### Speed up File Explorer for big folders
+
+For most users, upgrading to an SSD is sufficient for speeding up disk operations like browsing files. Still, even fast SSDs can halt to a crawl when browsing specific folders that contain a lot of files. If you are tired with these slowdowns, here's a simple trick to make browsing files faster for big folders.
+
+File Explorer in modern Windows versions has a system that tries to detect file types in each folder you open to optimize specific options, like default view. This mechanism can cause significant performance issues regardless of your storage type or its speed. Therefore, forcing Windows to "consider" each folder as one containing miscellaneous items should make things a lot faster.
+
+1. Search for "Registry Editor" in the Start menu.
+
+![Screenshot of the user searching for the Registry Editor in the Start menu search.](./img/useful-windows-tips/regedit-search-result.png)
+
```
Comment (sthivaios) [inline wiki/useful-windows-tips.md:215]:
```suggestion

::: danger Caution
Before making changes to the registry, you should always backup the current state of the folder you are modifying. Right click on the folder you are making the changes to, in the Registry Editor, then click **Export** and save the file somewhere safe. If something goes wrong, you can restore the backup by double-clicking on the file. You can also go to **File > Import**, in the Registry Editor.
:::

```

**Review by sthivaios — CHANGES_REQUESTED** [review]
Added the "backup the registry first" notes.

**Comment by itsWindows11** [comment]
PR should be ready to merge now, assuming there are no more changes to be requested.

**Review by Erisa — APPROVED** [review]
Still lgtm

**Pull request merged**